pkg/server/controller.go

```diff
@@ -219,6 +219,16 @@ func (ctrl *Controller) serverMux() (http.Handler, error) {
 		apiRouter.RegisterAdhocHandlers()
 	}
 
+	// FIXME: Honor ctrl.config.RemoteRead.Enabled.
+	authorizer := authz.NewAuthorizer(ctrl.log, httputils.NewDefaultHelper(ctrl.log))
+	appsRouter := apiRouter.PathPrefix("/apps").Subrouter()
+	appsRouter.Use(ctrl.authMiddleware(nil))
+
+	appsRouter.Path("").
+		Methods(http.MethodGet).Handler(ctrl.getAppsHandler())
+	appsRouter.Path("").
+		Methods(http.MethodDelete).Handler(authorizer.RequireAdminRole()(ctrl.deleteAppsHandler()))
```

Comment:
I think it can be simplified, taking into account that we don't need to specify empty path:
```go
appsRouter.Methods(http.MethodGet).Handler(ctrl.getAppsHandler())
appsRouter.Methods(http.MethodDelete).Handler(authorizer.RequireAdminRole()(ctrl.deleteAppsHandler()))
```

Reply:
:P leftover from my debugging